cmd/login.go

```diff
@@ -11,11 +11,17 @@ func getCmdLogin(gs *state.GlobalState) *cobra.Command {
 	loginCmd := &cobra.Command{
 		Use:   "login",
 		Short: "Authenticate with a service",
-		Long: `Authenticate with a service.
+		Long: `[deprecation notice]
+This command is deprecated and will be removed in a future release. Please use the "k6 cloud login" command instead.
+
+Authenticate with a service.
 
 Logging into a service changes the default when just "-o [type]" is passed with
 no parameters, you can always override the stored credentials by passing some
 on the commandline.`,
+		Deprecated: `This command is deprecated and will be removed in a future release.
+Please use the "k6 cloud login" command instead.
+`,
```

Comment:
By default, Cobra already prepends: `Command "login" is deprecated, `, see:

```
❯ go run main.go login -h
Command "login" is deprecated, This command is deprecated and will be removed in a future release.
Please use the "k6 cloud login" command instead.
```

so I'd suggest:

```suggestion
		Deprecated: `and will be removed in a future release.
Please use the "k6 cloud login" command instead.
`,
```